examples/example_material.xomp

```diff
@@ -15,6 +15,11 @@
         "surface_height_rms": 0.8,
         "surface_correlation_length": 1.0
     },
+    "emissivity_data": {
+        "emissivity": 0.07,
```

Comment:
```suggestion
        "emissivity_coefficient": 0.07,
```

Reply:
It was discussed in the material group, that emissivity_coefficient would be a bit more accurate.